### PR TITLE
Refactor: use `lsp-render-symbol`

### PR DESCRIPTION
**Pull request by nbfalcon**

### CODE REVIEW

**Comment by nbfalcon** [comment]
@yyoncho could you perhaps give me push access to this repository (not necessarily master), so I can create feature branches here instead of only in my fork?

File: lsp-treemacs.el [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ;; Author: Ivan Yonchovski
 ;; Keywords: languages
-;; Package-Requires: ((emacs "26.1") (dash "2.14.1") (dash-functional "2.14.1") (f "0.20.0") (ht "2.0") (treemacs "2.5") (lsp-mode "6.0"))
+;; Package-Requires: ((emacs "26.1") (dash "2.14.1") (dash-functional "2.14.1") (f "0.20.0") (ht "2.0") (treemacs "2.5") (lsp-mode "7.1"))
```
Comment (yyoncho) [inline lsp-treemacs.el:7]: revert  that

Comment (nbfalcon) [reply lsp-treemacs.el]: Fixed.

Comment (nbfalcon) [reply lsp-treemacs.el]: I didn't know that `lsp-mode` version dependencies don't mean anything, so this won't happen in the future.

Comment (yyoncho) [reply lsp-treemacs.el]: It means only for melpa-stable. I asked to revert since 7.1 is not released but in this case, actually, this bump was possible.

**Review by yyoncho — APPROVED** [review]

**Comment by nbfalcon** [comment]
You may want to close #76, since it conflicts with this (they decided to implement it manually, without `lsp-render-symbol`). `lsp-lens-face` is used a few times less because of this, but we should probably still consider not depending on `lsp-lens`, but using `lsp-details-face` instead.

**Pull request merged**